Frontend/src/cse-testing-lib/custom-queries.ts

```diff
@@ -0,0 +1,32 @@
+import {queryHelpers, buildQueries} from '@testing-library/react'
+
+// The queryAllByAttribute is a shortcut for attribute-based matchers
+// You can also use document.querySelector or a combination of existing
+// testing library utilities to find matching nodes for your query
+
+const queryAllByDataAnchor = (...args: any[]) => {
+  // @ts-ignore
+  return queryHelpers.queryAllByAttribute('data-anchor', ...args)
+}
```

Comment:
Hi, eslint complains about this function: eslint[@typescript-eslint/explicit-module-boundary-types] and eslint[@typescript-eslint/ban-ts-comment]
When I hover my mouse over the `buildQueries` function below I see react-testing-library type this function as `GetAllBy<[dataAnchorValue: any]>`. So I tried this and it gets rid of all the eslint errors:
```
import {queryHelpers, buildQueries, GetAllBy} from '@testing-library/react'

const queryAllByDataAnchor : GetAllBy<[dataAnchorValue: any]> = (...args) => {
  return queryHelpers.queryAllByAttribute('data-anchor', ...args)
}
```

Reply:
great pickup!! :D will update with this instead of the eslint ignore